source/extensions/filters/network/postgres_proxy/postgres_decoder.h

```diff
@@ -150,17 +160,20 @@ class DecoderImpl : public Decoder, Logger::Loggable<Logger::Id::filter> {
 
   // Helper method generating currently processed message in
   // displayable format.
-  const std::string genDebugMessage(const MessageProcessor& msg, Buffer::Instance& data,
+  const std::string genDebugMessage(const std::unique_ptr<Message>& parser, Buffer::Instance& data,
                                     uint32_t message_len);
 
   DecoderCallbacks* callbacks_{};
   PostgresSession session_{};
 
   // The following fields store result of message parsing.
-  char command_{};
+  char command_{'-'};
   std::string message_;
   uint32_t message_len_{};
 
+  static const std::string FRONTEND;
```

Comment:
static const string (non-POD) is source of flaky test, use singleton (CONSTRUCT_ON_FIRST_USE) or constexpr string_view.

Reply:
Converted to `constexpr string_view`. Thanks!